lib/galaxy/config/__init__.py

```diff
@@ -576,6 +576,21 @@ def _process_config(self, kwargs):
             if not self.file_path_set and self.file_path.endswith('objects'):
                 self.object_store_store_by = 'uuid'
         assert self.object_store_store_by in ['id', 'uuid'], "Invalid value for object_store_store_by [%s]" % self.object_store_store_by
+        self.object_store_cache_path = self.resolve_path(kwargs.get("object_store_cache_path", os.path.join(self.data_dir, "object_store_cache")))
+
+        # Configuration options for new storage media uses can plug-in.
+        self.enable_user_based_object_store = kwargs.get(
+            "enable_user_based_object_store", False)
+        self.default_storage_media_jobs_directory = self.resolve_path(kwargs.get(
```

Comment:
same as above

Reply:
also, you don't need to call `resolve_path` - that's already done as well

Reply:
Thanks.